stories/button.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Button from '../src/web/components/Button';
+import Icon from '../src/web/components/Icon';
+
+storiesOf('Button', module)
+  .add('basic button', () => <Button>Simple button</Button>)
+  .add('with size', () => [
+    <Button size="small">Simple button</Button>,
```

Comment:
With storybook, it's possible to put `actions` that will be log in the storybook console to verify a specific behaviour based on human events, such as a click.

For this component, I think that it would be great to log something `onClick`, like the button size for example.

I leave you check this document : 

https://github.com/storybooks/storybook/tree/master/addons/actions